devito/tools/abc.py

```diff
@@ -90,6 +90,10 @@ def _signature(self):
         return Signer._sign(self._signature_items())
 
 
+def wrapper(cls, args, kwargs):
```

Comment:
can this be a private method inside Pickable ?

Reply:
Maybe a classmethod - I'll try.